### PR TITLE
ssh as tsh ssh alias

### LINKED ISSUE #3728: Alias Example Does Not Match Description
## Summary
From the user guide:

For example, if you have the following alias defined in your ~/.bashrc: alias ssh="tsh ssh" then you can continue using familiar SSH syntax:

```
# Have this alias configured, perhaps via ~/.bashrc
$ alias ssh=/usr/local/bin/tsh
```

The command should be:

`$ alias ssh="/usr/local/bin/tsh ssh"`

as mentioned in the descriptive paragraph above.

### Audience
new users

### Location
an addition to an existing guide?

### PR DESCRIPTION
**Pull request by quinqu**
Fixed #3728

### CODE REVIEW

**Comment by webvictim** [comment]
I think that `tsh` is supposed to detect if it's called using the name `ssh` and modify its behaviour to automatically run `tsh ssh` in these instances. Does it work that way for you?

How about if you rename the binary (so `mv tsh ssh` or similar) - does it do that then?

**Comment by quinqu** [comment]
I think folks are confused by this line in the user manual, because they don't know that `ssh` is being appended to the command, I updated the description so it's less confusing. 

> alias ssh="tsh ssh" then you can continue using familiar SSH syntax

**Comment by webvictim** [comment]
Leaving this one for @benarent

File: docs/4.3/user-manual.md [hunk 1]
```diff
@@ -267,12 +267,12 @@ To launch an interactive shell on a remote node or to execute a command, use
 
 `tsh` tries to mimic the `ssh` experience as much as possible, so it supports
 the most popular `ssh` flags like `-p`, `-l` or `-L`. For example, if you have
-the following alias defined in your ~/.bashrc: `alias ssh="tsh ssh"` then you
+the following alias defined in your ~/.bashrc: `alias ssh="tsh"` then you
```
Comment (benarent) [inline docs/4.3/user-manual.md:270]:
```suggestion
the following alias defined in your ~/.bashrc: `alias ssh="tsh ssh"` then you
```

File: docs/4.3/user-manual.md [hunk 2]
```diff
@@ -267,12 +267,12 @@ To launch an interactive shell on a remote node or to execute a command, use
...
 can continue using familiar SSH syntax:
 
 ```bash
 # Have this alias configured, perhaps via ~/.bashrc
-$ alias ssh=/usr/local/bin/tsh
+$ alias ssh="/usr/local/bin/tsh"
```
Comment (benarent) [inline docs/4.3/user-manual.md:275]:
```suggestion
$ alias ssh="/usr/local/bin/tsh ssh"
```

**Review by benarent — CHANGES_REQUESTED** [review]
Based on the example and the initial issue, let's keep it to alias ssh to `tsh ssh` as the rest of the example shows this flow. 

Feel free to just commit my suggestions.

**Review by webvictim — APPROVED** [review]

**Comment by webvictim** [comment]
@benarent Let's get this closed/merged as appropriate.

**Review by benarent — APPROVED** [review]

**Comment by benarent** [comment]
ok to test

**Pull request merged**